pom.xml

```diff
@@ -3077,5 +3089,67 @@
     <profile>
       <id>sparkr</id>
     </profile>
+
+    <profile>
+      <id>thriftif</id>
```

Comment:
could this be in thriftserver pom file?

Reply:
> could this be in thriftserver pom file?

It is ok to move it to thriftserver pom.